packages/web-components/fast-foundation/docs/guide/styling-components.doc.md

```diff
@@ -3,4 +3,91 @@ id: styling-components
+
+## Design System
+In FAST, the *design system* is a collection of properties and values that inform the visual design language of the components. These properties are managed and provided through implementations of the *DesignSystemProvider*. See the [*DesignSystemProvider* documentation](fast-foundation/fast-design-system-provider.md) for more information.
+
+For the purposes of this section, the *DesignSystemProvider* is a Custom Element that will create CSS Custom Properties that can be consumed by component stylesheets and provide mechanisms to synchronize, update, and consume those properties programmatically.
+
+### Type Ramp
+The FAST type ramp is exposed by the `FASTDesignSystemProvider` as CSS Custom Properties. It centers the ramp around a _base_ font size and line-height, ascending and descending from the _base_. The CSS Custom Properties that can be used are:
+
+| Level              | Font Size                       | Line Height                      |
+|--------------------|---------------------------------|----------------------------------|
+| Minus 2 (smallest) | `--type-ramp-minus-2-font-size` | `--type-ramp-minus-2-line-height`|
+| Minus 1            | `--type-ramp-minus-1-font-size` | `--type-ramp-minus-1-line-height`|
+| Base (body)        | `--type-ramp-base-font-size`    | `--type-ramp-base-line-height`   |
+| Plus 1             | `--type-ramp-plus-1-font-size`  | `--type-ramp-plus-1-line-height` |
+| Plus 2             | `--type-ramp-plus-2-font-size`  | `--type-ramp-plus-2-line-height` |
+| Plus 3             | `--type-ramp-plus-3-font-size`  | `--type-ramp-plus-3-line-height` |
+| Plus 4             | `--type-ramp-plus-4-font-size`  | `--type-ramp-plus-4-line-height` |
+| Plus 5             | `--type-ramp-plus-5-font-size`  | `--type-ramp-plus-5-line-height` |
+| Plus 6 (largest)   | `--type-ramp-plus-6-font-size`  | `--type-ramp-plus-6-line-height` |
+
+### Design System Properties
+The following properties are provided by the `FASTDesignSystemProvider` and should be used as appropriate.
+
+| Name                                   | Type | Description   |
+|----------------------------------------|--------------------  |----------------------------------------------------------------------------|
+| `--background-color`                   | `string` (hex color) | Defines the background color of the node. This is used by color recipes and should represent the color UI is rendering on for a given node tree. |
+| `--density`                            | `number`             | A multiplier to control the density of UI elements.                        |
+| `--design-unit`                        | `number`             | The core sizing unit that all sizes are derived from.                      |
+| `--base-height-multiplier`             | `number`             | The number of designUnits used for component height at the base density.   |
```

Comment:
Is "designUnits" intended to be one word?  Is this a code term?  If so, maybe accentuate with italic, bold, or link to where we talk about designUnits?